docs/basics.rst

```diff
@@ -0,0 +1,103 @@
+Basic Syntax
+============
+
+Comments
+--------
+In :ref:`TL4 <syntax-tl4>` Single line comments start with ``;``, multi-line
```

Comment:
Don't you want to use a link directive that works both in readthedocs and in github's markdown (I haven't tested it but it looks like the links here: https://guides.github.com/features/mastering-markdown/ should work on both platforms).

Reply:
I will check this out...